src/Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -61,6 +61,10 @@ public CSharpKernel(string name) : base(name)
         KernelInfo.LanguageName = "C#";
         KernelInfo.LanguageVersion = "12.0";
         KernelInfo.DisplayName = $"{KernelInfo.LocalName} - C# Script";
+        KernelInfo.Description = """
+                                 This Kernel can compile and execute C# code and display the results.
+                                 The language is C# Scripting, a dialect of C# that is used for interactive programming.
+                                 """;
```

Comment:
Is this displayed anywhere in the UI? Will we need to localize this description for users who are on other languages?